src/main/java/sirius/biz/importer/txn/ImportTransactionHelper.java

```diff
@@ -68,14 +70,17 @@ public ImportTransactionHelper start() {
     /**
      * Starts a new transaction by setting the internal transaction id to the current timestamp and a source.
      *
-     * @param source the source, later used to delete unmarked items of the same source
+     * @param source     the source, later used to delete unmarked items of the same source
+     * @param deleteMode the mode used to delete unmarked items depending on the given source
      * @return the helper itself for fluent method calls
      */
-    public ImportTransactionHelper start(String source) {
+    public ImportTransactionHelper start(String source, SyncSourceDeleteMode deleteMode) {
         start();
-        if (Strings.isFilled(source)) {
+        // The source is never initialized with an empty string but null instead.
+        if (source != null && !source.isBlank()) {
```

Comment:
still using isFilled never hurts...

Reply:
Yeah, but isFilled() on a string with only white-spaces turns into `true`, and trimming it in an empty string.